expression/builtin_compare_vec_test.go

```diff
@@ -29,6 +29,7 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 	ast.Coalesce: {},
 	ast.NullEQ: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETReal, types.ETReal}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDecimal, types.ETDecimal}},
```

Comment:
If each decimal is randomly generated, the probability that such two decimals are equal will be very low. Can you add some cases in which you can use some new ```dataGener```(for example ```rangeDecimalGener```) to generate decimal? It can increase the chances of generating the same decimal which can increase the possibility of two decimals being equal, to make the test more comprehensive.

Reply:
I couldn't find `rangeDecimalGener`, so I used
 `gener {defaultGener {eType: types. ETDecimal, nullRation: 0.2}}` 
to generate test case.